packages/@okta/vuepress-site/docs/concepts/oauth-claims/index.md

```diff
@@ -0,0 +1,125 @@
+
+> **Note:** Access tokens are sometimes formatted as JWTs, but not always.
+
+### Claims versus scopes
+
+Claims and scopes are related in OAuth 2.0 and OIDC, but have some important differences. [Scopes](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#scopes) are used to request access to specific resources or actions, while [claims](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#claims) are used to provide information about users and their permissions.
+
+During the authorization flow, an app requests specific scopes. The resulting access token or ID token includes claims that correspond to those scopes. The claims are the actual data returned as a result of that request.
+
+You can use claims for fine-grained permissions and information to enhance the security of your apps. Scopes include bundles of claims, whereas claims provide more granularity that allows you to control access to specific resources or actions.
+
+### Claims in access tokens
+
+Claims in access tokens pass information about the user and their permissions to the resource server. Claims are contained within a scope, or you can [add custom claims](/docs/guides/customize-tokens-returned-from-okta/main/) to an access token. Access tokens can contain scopes and custom claims, while ID tokens can contain claims, but not scopes.
+
+Access and ID tokens contain a [header](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#access-token-header), [payload](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#access-token-payload), and [signature](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#access-token-signature).
+
+Along with other parameters, the payload can contain scopes and custom claims. See the following example of an access token payload.
+
+```json
+{
+  "ver": 1,
+  "jti": "AT.0mP4JKAZX1iACIT4vbEDF7LpvDVjxypPMf0D7uX39RE",
+  "iss": "https://{yourOktaDomain}/oauth2/{authorizationServerId}",
+  "aud": "https://api.example.com",
+  "sub": "00ujmkLgagxeRrAg20g3",
+  "iat": 1467145094,
+  "exp": 1467148694,
+  "cid": "nmdP1fcyvdVO11AL7ECm",
+  "uid": "00ujmkLgagxeRrAg20g3",
+  "scp": [
+    "openid",
+    "email",
+  ],
+  "auth_time": 1467142021,
+}
+```
+
+The payload contains various [reserved claims](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#reserved-claims-in-the-payload-section), along with an array of scopes, within the `scp` array. The reserved claims convey some information about the access token. For example, the `iss` (issuer) claim contains the URL of the authorization server that issued the token. And the `aud` (audience) claim contains the intended recipient of the token.
```

Comment:
Might want to add a sentence here that you're talking about the example above, like "in the access token example, the payload contains....."

Because when I got to the next paragraph, you said the `scp` parameter passes two scopes to the resource server:  but it can pass more than these two. It could be misleading to newbies.